src/store/wallet.js

```diff
@@ -65,6 +68,11 @@ export default {
         }
       })
     },
+    setWalletIndex(state, { walletIndex }) {
+      let index = walletIndex
+      walletIndex === -1 ? (index = 0) : (index = walletIndex)
+      state.walletIdx = index
+    },
```

Comment:
I think this could be equivalent:

```suggestion
    setWalletIndex(state, { walletIndex }) {
      state.walletIdx = walletIndex === -1 ? 0 : walletIndex;
    },
```


Or even clearer and safer (will change to `0` any negative values):

```suggestion
    setWalletIndex(state, { walletIndex }) {
      state.walletIdx = Math.max(0, walletIndex);
    },
```